tests/cases_catalogs.py

```diff
@@ -79,15 +79,15 @@
     pd.DataFrame(
         {
             "map_id_candidate": [
-                "s3://gval-test/candidate_continuous_0.tif",
-                "s3://gval-test/candidate_continuous_1.tif",
-                "s3://gval-test/candidate_continuous_1.tif",
+                f"{TEST_DATA_DIR}/candidate_continuous_0.tif",
```

Comment:
Yes it would, I don't know if this was done for memory reasons  in the pipeline tests but the original creation of this script used the pattern so I emulated it.  I can refactor that for this PR if you prefer.

Reply:
Your original implementation is comparing the agreement catalog fields which are strings in this case and does not necessitate actually loading them as xarray objects since any loading is being done in the catalog_compare method.  If there is a desire to change this testing we should put this in a refactor PR.